contract_comm/blockchain_parameters/blockchain_parameters.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 The Celo Authors
```

Comment:
Per a suggestion from @timmoreton, consider adding a flag that allows users to disable this check. This will allow the network to recover more quickly in the event that a malicious actor is able to manipulate the minimum client version